### DISCUSSION TITLE
浅色主题下代码高亮问题

### DISCUSSION THREAD

**Discussion by abowloflrf** [geeknote-net/feedback#13] [General]
如图～

<img width="826" alt="image" src="https://user-images.githubusercontent.com/17028350/160243292-35ced4f9-a50b-4c98-92c0-2139b8784e84.png">

<img width="914" alt="image" src="https://user-images.githubusercontent.com/17028350/160243305-7a0a2489-3886-4511-88a1-046e5a147660.png">

**Comment by chloerei** [comment]
是什么浏览器？

**Reply by chloerei** [reply]
我用机子上有的浏览器无法重新，如果方便的话帮调试下样式，看看无法显示的内容的字体颜色是什么值？

<img width="493" alt="截屏2022-03-26 22 21 12" src="https://user-images.githubusercontent.com/49931/160243718-d5e775d9-b2cf-4450-9041-ea0117c48c13.png">

样式表拉下一点，看是哪个变量集合在生效？

<img width="492" alt="截屏2022-03-26 22 22 18" src="https://user-images.githubusercontent.com/49931/160243750-5a99885b-48ec-47ec-81b8-6affef51533d.png">

**Reply by chloerei** [reply]
我发现问题了，系统深色调，主题调浅色调的时候会出现，我尽快修复。

**Reply by abowloflrf** [reply]
MacOS 上 Edge & Safari 均有这个问题

<img width="1919" alt="image" src="https://user-images.githubusercontent.com/17028350/160243901-51d8eae4-39f8-4cfa-997c-6422116ff2cc.png">

**Reply by chloerei** [reply]
修复了，可以再看看。

**Reply by abowloflrf** [reply]
嗯现在看正常了
